features/environment.py

```diff
@@ -412,6 +412,59 @@ def before_scenario(context: Context, scenario: Scenario):
             )
 
 
+def after_scenario(context, scenario):
+    """Collect the coverage files after the scenario is run."""
+    cov_dir = os.path.join(context.pro_config.artifact_dir, "coverage")
+    if not os.path.exists(cov_dir):
+        os.makedirs(cov_dir)
+
+    inner_dir = os.path.join(
+        datetime.datetime.now().strftime("%Y-%m-%d"),
```

Comment:
We may not actually need the directory with a date name. 
Our unit tests just overwrite the existing .coverage file if there is one.
What do you think?

Reply:
I feel instead of the date folder, we can just have the coverage folder which has the `.coverage` for each feature and we can collect it from there.

The date folder was there just in case comparing coverage was ever needed, so it can be removed